src/System.Data.SqlClient/tests/ManualTests/SQL/SqlBulkCopyTest/AzureDistributedTransaction.cs

```diff
@@ -0,0 +1,70 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+using System.Transactions;
+using Xunit;
+
+namespace System.Data.SqlClient.ManualTesting.Tests
+{
+    public class AzureDistributedTransaction
+    {
+        private static readonly string s_connectionString = DataTestUtility.TcpConnStr;
+        private static readonly string s_tableName        = "Azure_" + Guid.NewGuid().ToString().Replace('-', '_');
+        private static readonly string s_createTableCmd   = $"CREATE TABLE {s_tableName} (NAME NVARCHAR(40), AGE INT)";
+        private static readonly string s_sqlBulkCopyCmd   = "SELECT * FROM(VALUES ('Fuller', 33), ('Davon', 49)) AS q (FirstName, Age)";
+        private static readonly int    s_commandTimeout   = 30;
+
+        public static void Test()
+        {
+            try
+            {
+#if DEBUG
+               Console.WriteLine($"Creating Table {s_tableName}");
```

Comment:
Debug.WriteLine?

Reply:
It seems like `Debug.WriteLine` writes to Debug tab in the Output window in the Visual Studio and not the Console Window where the test is running, will keep the current changes.